### ISSUE TITLE
v2 change two state 

### ISSUE THREAD

**Issue by devmsh** [spatie/laravel-model-states#144]
After upgrade to v2, when I try to transition twice in the same request it give me

```php
Error : Typed property Spatie\ModelStates\State::$field must not be accessed before initialization
 /Users/devmsh/projects/backend/vendor/spatie/laravel-model-states/src/State.php:233
 /Users/devmsh/projects/backend/vendor/spatie/laravel-model-states/src/State.php:140
```

**Comment by lorenzolosa** [comment]
I got the same error. The problem is that transition() do not set the $field property when creating a new state. I've written a pull request for this in #145.

**Comment by brendt** [comment]
Fixed in 2.0.1.
